model/src/main/java/com/epam/esm/entity/Certificate.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.esm.entity;
+
+import javax.persistence.*;
+import javax.persistence.Entity;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "certificates")
+public class Certificate extends com.epam.esm.entity.Entity<Long> {
+
+    @Column(name = "name")
```

Comment:
sorry for resolving conversation. I think it is better to use @Column with indication name of column, because it is will be happen that field of class will changed or name of column in table will changed and all of this will result in an error syncing row from table and object and it's easier to find the error if all annotations are specified (@Column with name)

Reply:
No objections.